Library/Homebrew/cask/lib/hbc/cli/info.rb

```diff
@@ -48,9 +48,11 @@ def self.name_info(cask)
     puts cask.name.empty? ? "#{Tty.red}None#{Tty.reset}" : cask.name
   end
 
-  def self.github_info(cask)
+  def self.repo_info(cask)
     user, repo, token = Hbc::QualifiedToken.parse(Hbc.all_tokens.detect { |t| t.split("/").last == cask.token })
-    "#{Tap.fetch(user, repo).default_remote}/blob/master/Casks/#{token}.rb"
+    remote = Tap.fetch(user, repo).remote
+    return "#{remote}/blob/master/Casks/#{token}.rb" if remote.include? "github.com"
```

Comment:
`include?` here fails because `remote` returns `nil` if the tap is untapped. I'd suggest falling back to `default_remote` in this case.